examples/service-catalog/service-catalog.yaml

```diff
@@ -0,0 +1,411 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
+- kind: ServiceAccount
```

Comment:
(nice to have) would it be possible to have a named SA for the api server as well?